streams.yml

```diff
@@ -74,7 +74,7 @@ rhel-7-golang:
 rhel8:
   # the most recent release at present. since we yum update this, maybe it does not need to float.
   # but it is important that we not build from unreleased builds.
-  image: registry-proxy.engineering.redhat.com/rh-osbs/ubi8:8.4-213
+  image: registry-proxy.engineering.redhat.com/rh-osbs/ubi8:8.5
```

Comment:
i know i said we could track latest, but i realized i'm wrong... if we do that, we'll be immediately publishing these as they're built, before RHEL can ship them.

so we should pin to what's shipped, currently 8.5-236. we can re-pin once we switch to 8.6 beta.